test/jit/test_symbolic_shape_analysis.py

```diff
@@ -96,6 +104,16 @@ def apply(fn):
             torch._C._jit_pass_propagate_shapes_on_graph(t.graph)
             self.assertEqual(next(t.graph.outputs()).type().symbolic_sizes(), [2, 2])
 
+    @skipIfNoTorchVision
```

Comment:
this test is weirdly failing on just a couple platforms, and it doesnt actually test shape engine because output will always be [1, 1000] regardless of inputs so im removing it

Reply:
although it was useful to verify by looking at graph